packages/prompts/src/index.ts

```diff
@@ -191,6 +192,48 @@ export const select = <Options extends Option<Value>[], Value extends Primitive>
 	}).prompt() as Promise<Options[number]['value'] | symbol>;
 };
 
+export const selectKey = <Options extends Option<Value>[], Value extends string>(
+	opts: SelectOptions<Options, Value>
+) => {
+	const opt = (
```

Comment:
I know this isn't new and follow the style rest of the code, but I prefer to spell out words. I.e. `opts` -> `options` and `opt` -> `option`. 

Any reason not to destruct the options? As in `{options, initialValue}` 

I don't find `opt` to be a descriptive name. I would prefer something along the lines `render/printOption`.